app/lib/brexit_checker/actions.yaml

```diff
@@ -1699,7 +1700,9 @@ actions:
   guidance_link_text: Traffic management measures for Dover and Eurotunnel
   guidance_url: https://www.gov.uk/guidance/kent-traffic-management-on-m20-motorway-to-dover-and-eurotunnel
   criteria:
-  - haulage-goods-across-eu-borders
+  - any_of:
+    - haulage-goods-across-eu-borders
+    - road-passenger-freight
```

Comment:
Just to confirm: these two criteria changes don't need a notification?

Reply:
Ah sorry Ben, they were committed in error. I've removed since your comment so that they can be done in a separate PR, with their notifications.